docs/db2graph/db2graph.rst

```diff
@@ -0,0 +1,279 @@
+         ...
+        (30 rows)    
+   
+   | This creates 15 tables containing information about actors, movies, keywords, production companies, production countries, as well as credits data.
+   
+   | Install Marius and the required dependencies for Db2Graph.
+   
+   .. code-block:: bash 
+       
+       $ cd / # back to root directory
+       $ apt-get update
+       $ apt-get install vim
+       $ apt-get install git
+       $ apt-get install python3
+       $ apt-get install python3-pip
+       $ pip3 install psycopg2-binary
+       $ pip3 install mysql-connector-python
+       $ pip3 install hydra-core --upgrade
+       $ git clone https://github.com/marius-team/marius.git
+       $ pip3 install. 
+
+#. | Next, create the configuration files. From the root directory, navigate to the Db2Graph directory and create the ``conf/config.yaml``, ``conf/entity_nodes.txt``, ``conf/edges_entity_entity.txt``, and ``conf/edges_entity_feature_values.txt`` files if they have not been created. 
+
+    .. code-block:: bash 
+       
+       $ cd Marius/src/python/tools/db2graph/
+       $ vi conf/config.yaml
+
+   | In ``conf/config.yaml``, define the following fields:
+    
+    .. code-block:: yaml
+        
+            db_server: postgre-sql
+            db_name: test_db
+            db_user: postgres
+            db_password: password
+            db_host: 127.0.0.1
+            generate_uuid: false 
+            entity_node_queries: conf/entity_nodes.txt
+            edges_entity_entity_queries: conf/edges_entity_entity.txt
+            edges_entity_feature_values_queries: conf/edges_entity_feature_values.txt
+
+   | In ``conf/edges_entity_entity.txt``, define the following queries. Note that we create three edges/relationships: An actor acted in a movie; A movie directed by a director; A movie produced by a production company.
+    
+    .. code-block:: sql
+           
+           acted_in
+           SELECT persons.name, movies.title FROM persons, actors, movies WHERE persons.id = actors.person_id AND actors.movie_id = movies.id;
+           directed_by
+           SELECT movies.title, persons.name FROM persons, directors, movies WHERE persons.id = directors.director_id AND directors.movie_id = movies.id;
+           produced_by
+           SELECT movies.title, production_companies.name FROM production_companies, movies_production_companies, movies WHERE production_companies.id = movies_production_companies.production_company_id AND movies_production_companies.movie_id = movies.id;  
+
+   | For simplicity, we limit the queries to focus on the movies table. The user can expand or shorten the list of queries in each of the above query definition files to query a certain subset of data from the database.
+
+#. | Lastly, execute Db2Graph with the following script:
+
+    .. code-block:: bash
+        
+           $ python3 db2graph.py --config_path conf/config.yaml --output_directory output_dir/
```

Comment:
Update this to use the installed command `marius_db2graph` instead

Reply:
Updated